diff --git a/package.json b/package.json
index 25bc7b48d045ddba238225bd7845a4fcd5aaefc0..c6684acc48806d9258a254d94a0e190442a849ff 100644
--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "name": "storybook-react-router",
   "version": "1.0.8",
   "description": "A storybook decorator that allows you to use routing-aware components in your stories.",
-  "main": "./dist/react.js",
+  "main": "./react.js",
   "license": "MIT",
   "author": "Gianni Valdambrini",
   "repository": {
diff --git a/react.js b/react.js
index e744e8a11137384df1c9dde833c9f0e784018bb4..783d00bf6d308f77afb7c22f78619ea4cb0795aa 100644
--- a/react.js
+++ b/react.js
@@ -2,7 +2,7 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 
 import { action } from '@storybook/addon-actions';
-import { MemoryRouter, matchPath, Route } from 'react-router';
+import { MemoryRouter, matchPath, Route } from 'react-router-dom';
 
 const match = (link, path) => {
   // If the new path matches with one of the keys defined in the links object, then
